app/screens/index.js

```diff
@@ -14,6 +14,7 @@ import SelectServer from 'app/screens/select_server';
 const navigator = {
     push: () => {}, // eslint-disable-line no-empty-function
     setOnNavigatorEvent: () => {}, // eslint-disable-line no-empty-function
+    setStyle: () => {}, // eslint-disable-line no-empty-function
```

Comment:
nothing to do here. I'm assuming this object will get deleted after the migration is completed

Reply:
Yep, there's a comment a few lines up explaining that